lib/wor/paginate/paginate.rb

```diff
@@ -12,6 +12,10 @@ module Paginate
     ].freeze
 
     def render_paginated(content, options = {})
+      if includes?(options)
```

Comment:
can you make it in a single line this i think 🤔 ...  try to avoid the `if - end` thing
if its too long you could use another method
```
return render_paginate_with_include(content, options) if  includes?(options)
...
end

def render_paginate_with_include(content, options) 
  return render json: paginate(content, options), include: options[:include]
end
```